django/core/management/commands/inspectdb.py

```diff
@@ -232,7 +232,9 @@ def get_field_type(self, connection, table_name, row):
             field_type = connection.introspection.get_field_type(row[1], row)
         except KeyError:
             field_type = 'TextField'
-            field_notes.append('This field type is a guess.')
+            field_notes.append(
+                'This field type is a guess. It may be supported by contrib or third party apps.'
```

Comment:
My inclination is to omit this change as part of this commit. For new users, I think it will raise more questions than answers. I think it's probably better to have some more specific instructions in `docs/howto/legacy-databases.txt` about it. Considering there's no stable API for it, I'd be surprised if there are any third-party apps that are adding introspection for their fields but if there are some examples, I'd be interested to know about it.